ai_diffusion/ui/autocomplete.py

```diff
@@ -0,0 +1,243 @@
+from enum import Enum
+from dataclasses import dataclass
+import csv
+from pathlib import Path
+from typing import List
+
+from PyQt5.QtWidgets import QCompleter, QLineEdit, QStyledItemDelegate, QStyle
+from PyQt5.QtGui import QFont, QPainter, QPen, QColor, QFontMetrics
+from PyQt5.QtCore import Qt, QStringListModel, QSize, QRect, QAbstractProxyModel
+
+from ..root import root
+from ..settings import settings
+from ..text import LoraId
+from ..util import ensure, plugin_dir, user_data_dir
+
+
+class TagType(Enum):
+    General = 0
```

Comment:
For consistency: use lower case for enum tags

Reply:
done